src/discounts/components/VoucherLimits/VoucherLimits.tsx

```diff
@@ -72,6 +72,15 @@ const VoucherLimits = ({
           name={"applyOncePerCustomer" as keyof VoucherDetailsPageFormData}
           onChange={onChange}
         />
+        <ControlledCheckbox
+          checked={data.onlyForStaff}
+          label={intl.formatMessage({
```

Comment:
Since we decided to put messages in a separate file, maybe this is a good moment to do that?

Reply:
sure, done